plugins/woocommerce/includes/abstracts/abstract-wc-order.php

```diff
@@ -2605,4 +2605,34 @@ public function set_cogs_total_value( float $value ) {
 			$this->set_prop( 'cogs_total_value', $value );
 		}
 	}
+
+	/**
+	 * Return the HTML to render the total Cost of Goods Sold for the order.
+	 *
+	 * @param array|null $wc_price_arg Arguments to be passed to wc_price, defaults to an array containing only the currency symbol.
+	 * @return string
+	 */
+	public function get_cogs_total_value_html( ?array $wc_price_arg = null ): string {
+		if ( ! $this->cogs_is_enabled( __METHOD__ ) || ! $this->has_cogs() ) {
```

Comment:
Micro-optimization opportunity: changing the conditions order as `has_cogs` does less.

Reply:
The thing is that `cogs_is_enabled` triggers a `doing_it_wrong` if the feature isn't enabled. I think it's convenient to loudly "shout" that this method is being incorrectly used, even in the cases where the specific order class doesn't handle COGS (also our own code isn't supposed to call this method if COGS is disabled, so we want to know if that is not the case).

Reply:
That makes sense to me, marking the thread as resolved.